profile/README.md

```diff
@@ -66,8 +66,10 @@
 [revanced-integrations]: https://contrib.rocks/image?repo=revanced/revanced-integrations&max=12
 [revanced-manager]: https://contrib.rocks/image?repo=revanced/revanced-manager&max=12
 [revanced-website]: https://contrib.rocks/image?repo=revanced/revanced-website&max=12
+[revanced-GmsCore]: https://contrib.rocks/image?repo=revanced/gmscore&max=12
```

Comment:
It is not branded as ReVanced GmsCore, just GmsCore

Reply:
Fixed both in latest commit.